### DISCUSSION TITLE
The main thread is stuck when calling Skia.SoftwareRenderSurface.OnDraw

### DISCUSSION THREAD

**Discussion by LionelXN** [unoplatform/uno#11360] [Q&A]
Some times，i used "Surface.OnDraw", it would run out of the following exception，m I using it incorrectly, or what is the potential hidden danger?

 protected override void OnDraw(SKCanvas canvas)
 {
     _path.Reset();
     canvas.DrawPath(_path, _paint);
 }
![image](https://user-images.githubusercontent.com/18021306/219632390-5e2df94d-d874-4c81-a39f-d63acb72b0a7.png)

**Comment by jeromelaban** [comment]
Thanks for the report. 

For all the discussions you are creating, we'll ask you for repro samples. Make sure to provide one so we can discuss further the issues you are reporting.
